std_msgs/Header header
common/FloatTimeStamped cdcu_rmtchn3_freq
common/FloatTimeStamped cdcu_rmtchn3_duty
common/FloatTimeStamped cdcu_rmtchn4_freq
common/FloatTimeStamped cdcu_rmtchn4_duty